### ISSUE TITLE
Heavy Performance Impact - Extend Rope

### ISSUE THREAD

**Issue by Staynex** [sethduda/AdvancedSlingLoading#4]
Very nice script! We really like it! 
Only one problem - if you extend the rope like 10 times or more (its unlimited by default) your FPS will drop sharply :/ maybe only in Multiplayer. I disabled the feature to extend and shorten ropes. Because our users trolling with this -  they fly a heli with 7895768465 meters of ropes in a town and the FPS are like 5 ^^ If you retract the ropes, everything works smooth again!

**Comment by sethduda** [comment]
Yeah, I think the rope simulation in arma is pretty resource intensive. I can implement a max rope length.
